src/common/utils.ts

```diff
@@ -479,6 +479,7 @@ export function downloadFile(data: any, fileName: string, prefix?: string): void
 export function  getTagCategory (tagType: string) {
     switch (tagType) {
         case FieldType.SelectionMark:
+            return FieldType.SelectionMark;
```

Comment:
suggestion: an other thoughts for fixing that is to modify https://github.com/microsoft/OCR-Form-Tools/blob/e0d99d220d5ee08ed0a17dde02d93df20f7739c4/src/react/components/common/tagInput/tagInput.tsx#L592, can you take a look? thanks!